applications/conceptual-model-editor/src/action/actions-react-binding.tsx

```diff
@@ -57,6 +66,8 @@ interface DialogActions {
 
 interface VisualModelActions {
 
+  addSemanticEntityToVisualModel: (identifier: string, position?: { x: number, y: number } | null) => void;
+
   addClassToVisualModel: (model: string, identifier: string, position: { x: number, y: number } | null) => void;
```

Comment:
I would keep the null, at least for now, since I implemented the "clever" positioning of newly placed classes using layouting, so keep it at least until we get to PR of that branch.